atlasdb-workload-server-api/src/main/java/com/palantir/atlasdb/workload/resource/CassandraResource.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.workload.resource;
+
+public interface CassandraResource {
+    /**
+     * Flushes all keyspaces on a Cassandra host.
+     */
+    void flush(String host);
+
+    /**
+     * Performs a major compaction for all keyspaces on Cassandra.
```

Comment:
We should probably leave a note somewhere that these methods are non-blocking, for future reference (since we actually just log `antithesis: compact cassandra1` and don't actually wait for these to happen)